### DISCUSSION TITLE
Require util module in migration

### DISCUSSION THREAD

**Discussion by bmingles** [pocketbase/pocketbase#3898] [Q&A] [answered]
Is it possible to require a shared util module in migrations? Things I've tried

- Include the util module in the pb_migrations folder. This doesn't work since pocketbase tries to use it as a migration
- Include the util module in pb_hooks using ``require(`${__hooks}/config.js`)``. Get an error "ReferenceError: __hooks is not defined at <eval>"

**Comment by ganigeorgiev** [comment, marked as answer]
It should work if you specify an absolute path to the file (or relative to the executable).

I'll consider exposing the `__hooks` variable in the migrations loader with the next release.

**Reply by bmingles** [reply]
Relative path to executable works. Thanks.
